app/javascript/components/create-board/create-board.tsx

```diff
@@ -42,18 +75,18 @@ export const CreateBoard = ({isCreateBoardOpen, setCreateBoardOpen}) => {
     }
   });
 
-  const [emojiData, setEmojiData] = useState({
+  const [emojiData, setEmojiData] = useState<{ emojiColumnName: string, emojiPickerOpen: boolean }>({
     emojiColumnName: '',
     emojiPickerOpen: false
   });
 
-  const handleSubmit = (event) => {
+  const handleSubmit = (event: any) => {  //!!!! type
     alert(JSON.stringify(formData));
     event.preventDefault();
     setCreateBoardOpen(false);
   };
 
-  const onEmojiClick = (name) => (event, emojiObject) => {
+  const onEmojiClick = (name: string) => (event: React.MouseEvent<Element, MouseEvent>, emojiObject: IEmojiData) => {
```

Comment:
```suggestion
  const onEmojiClick = (name: string) => (event: MouseEvent, emojiObject: IEmojiData) => {
```

onEmojiClick on line 105 gives error and wants event: MouseEvent, do you have the same?

Reply:
No, I do not have this error